server/src/main/java/io/spine/server/BoundedContextBuilder.java

```diff
@@ -247,8 +253,12 @@ public BoundedContext build() {
     }
 
     private BoundedContext buildDefault(SystemBoundedContext system) {
-        BoundedContext result =
-                buildPartial(builder -> DomainBoundedContext.newInstance(builder, system));
+        BiFunction<BoundedContextBuilder, SystemGateway, DomainBoundedContext> instanceFactory =
+                (builder, systemGateway) -> DomainBoundedContext.newInstance(builder,
```

Comment:
This block of changes seems odd. In particular the `buildPartial()` and the fact that both system bounded context _and_ gateway are used to create a `DomainBoundedContext` instance.

Let's discuss in person.